src/index.test.js

```diff
@@ -553,6 +553,26 @@ test('Gives up retrying on API rate limiting after a timeout', async t => {
   t.false(scope.isDone())
 })
 
+test('Recreates a function body when handling API rate limiting', async t => {
+  const deploy_id = '2'
```

Comment:
```suggestion
  const deploy_id = '3'
```

So it does not collide with the other test with the same `deploy_id`. Tests are run in parallel.